### DISCUSSION TITLE
Option to hide the side menu bar on the left

### DISCUSSION THREAD

**Discussion by sassanix** [FreshRSS/FreshRSS#7099] [Ideas]
It would be nice to have the option to hide the sidebar with a click, and then allow it to come back. 

It would make it easier to read in portrait mode.

**Comment by math-GH** [comment]
Follow up here: https://github.com/FreshRSS/FreshRSS/issues/7100

**Comment by lukasMega** [comment]
It could be as part of the core - I think.
I remember it was the first thing I searched when I installed FreshRSS and I found extension for it:
- https://framagit.org/nicofrand/xextension-togglablemenu

screenshot from extension options:

<img width="666" alt="scr 2024-12-13 at 16 44 11" src="https://github.com/user-attachments/assets/cdf44bb5-659c-4132-ac1f-df1dac4eeac1" />

**Reply by sassanix** [reply]
It only works in the reading view and not in the normal view.
